libs/agno/agno/playground/playground.py

```diff
@@ -26,23 +34,42 @@ def __init__(
         settings: Optional[PlaygroundSettings] = None,
         api_app: Optional[FastAPI] = None,
         router: Optional[APIRouter] = None,
+        app_id: Optional[str] = None,
+        name: Optional[str] = None,
+        description: Optional[str] = None,
+        monitoring: bool = False,
```

Comment:
I don't think this should be optional. If you make a playground app, it would by default be tracked in the agno platform